raiden/network/proxies/token_network_registry.py

```diff
@@ -236,3 +236,9 @@ def get_token_network_created(self, to_block: BlockSpecification = "latest") ->
         return self.proxy.contract.functions.token_network_created().call(
             block_identifier=to_block
         )
+
+    def get_max_token_networks(self, to_block: BlockSpecification = "latest") -> int:
+        """ Returns the maximal number of TokenNetwork contracts that the
+        token network registry.
+        """
+        return self.proxy.contract.functions.max_token_networks().call(block_identifier=to_block)
```

Comment:
Maybe we can cache this value, says @hackaugusto .

Reply:
There are floating ideas to make TokenNetworkRegistry more flexible, so I keep it as it is.